doc/script_commands.txt

```diff
@@ -10403,3 +10405,16 @@ This value always smaller or equal to MAX_INVENTORY.
 Size can be changed by group of functions expandInventory*
 
 ---------------------------------------
+
+*getunittitle(<bl_id>)
```

Comment:
```suggestion
*getunittitle(<GID>)
```

```
*setunitdata(<GID>, <DataType>, <Val>)
....
*unitwalk(<GID>, <x>, <y>)
*unitwalk(<GID>, <target_GID>)
....
*unitkill(<GID>)
*unitwarp(<GID>, <Mapname>, <x>, <y>)
*unitattack(<GID>, <Target ID>)
*unitstop(<GID>)
*unittalk(<GID>, <Text>{, show_name{, <send_target>{, <target_id>}}})
*unitemote(<GID>, <Emote>)
````

Reply:
or and i not add packet version in documentation. for older versions it totally invisible